### DISCUSSION TITLE
R68S R22.7.1 国际版固件下载

### DISCUSSION THREAD

**Discussion by coolsnowwolf** [coolsnowwolf/lede#9539] [R68S / 电犀牛专区]
R22.7.1 下载：
加入 Flow 加速支持，避免jdc 收益低的问题，和 SFE 2选1

无Docker版本下载：
链接: https://pan.baidu.com/s/1NmEJgakuMhb1nc46u4M6ng 提取码: nyav 
Docker 版本下载：
链接: https://pan.baidu.com/s/1xbmFgqcs5AY5ba8rzTrWCA 提取码: mgn8

----------------------------
R22.6.26 下载：

1. 修复带宽监控 APP 的缓存默认值太小的问题
2. 修复 docker 和 UDP 中继的转发失效问题
3. 修复 USB 复制文件时偶然速度下降的问题
4. 修复 RTL USB 2.5G 网卡不识别的问题
5. 在局域网有华为/ZTE 某些 LAN/WAN 可混用的 AP 上强制开启 DHCP

无Docker版本下载：
链接: https://pan.baidu.com/s/1I6DBltMzFTluj21mIANAgA 提取码: hmdb 
Docker 版本下载：
链接: https://pan.baidu.com/s/1jF03gAn8z-_VWfzfKrt3ng 提取码: r3ax 

---------------------------------

R22.6.15下载：

1. 增加了  IPv6 的 luci 显示
6. 增加了 kmod-fuse 支持 aliyundrive-fuse 
7. 增加了 pptp 相关的 kmod 内核支持
8. 增加了 kmod-ipt-nat6 的 NAT6 的支持
9. 增加了 NTFS3 开机自动挂载的支持
10. 增加了一些 USB 网口的支持，例如 AIX 或者 RTL 2.5 G 的
11. 默认内置主题 Argon，可以手动直接启用

无Docker 版本下载:
链接: https://pan.baidu.com/s/13it6wbEYYt27a2kJKsSsHA 提取码: y8pc 
R22.6.15 Docker 版本下载：
链接: https://pan.baidu.com/s/1VzNgKSRyPvVbXTduHC0Yhg 提取码: a5ss 

------------------------------------
R22.6.11 下载: 
链接: https://pan.baidu.com/s/1vsE_Fl4jIUpRxiaJN1_Mbw 提取码: maae

**Comment by coolsnowwolf** [comment]
增加了 kmod-usb-net kmod-usb-net-rndis kmod-usb-net-cdc-ether kmod-fuse

**Reply by Suyuan406** [reply]
大雕可以基于这个版本做一个有docker的吗，自己去软件包装的docker没有web界面

**Reply by kiong2016** [reply]
我也遇到同样问题，提示没有dockerd文件，软件源里面也没有dockerd安装包！

**Reply by Suyuan406** [reply]
可以单独提供这四个模块吗，想在有docker版本下面的安装
kmod-usb-net kmod-usb-net-rndis kmod-usb-net-cdc-ether kmod-fuse

**Reply by disorderman** [reply]
求增加kmod-ebtables、kmod-ebtables-ipv4这两个包

**Comment by disorderman** [comment]
需要kmod-ebtables相关的包，不知能否提供，其他地方的对应的kernel版本都对不上

**Comment by smeor** [comment]
> 链接: https://pan.baidu.com/s/1vsE_Fl4jIUpRxiaJN1_Mbw 提取码: maae 
大雕啊，万恶得百度能不能换个

**Reply by xiehuijie** [reply]
本人搭建了一个r68s的固件站，欢迎访问
链接：https://r68s.geek213.fun

**Reply by copperfieldd** [reply]
阿里云好评，不咋限速

**Reply by xaqingfeng** [reply]
很不错，只是不能同步更新。另外能不能补充一些软件源的地址进来

**Comment by wfh-cool** [comment]
![image](https://user-images.githubusercontent.com/52191722/173805433-c0bb571e-8692-4af0-bb64-961a88442643.png)
alist无法启动

**Reply by YuyangHou1230** [reply]
我也是这个问题

**Reply by shaufo** [reply]
我也是这个问题

**Reply by yuyi4511** [reply]
我也是，同时自己安装插件，如果需要才创建目录的话，就启动不了。
日志如下：
1、filebrowser 
2022-06-16 15:04:51: /tmp/filebrowser not exists, please download first
2、阿里云 fuse
Error: No such file or directory (os error 2)

**Comment by ssmmtt** [comment]
![image](https://user-images.githubusercontent.com/16244320/173829992-e6b4d442-6428-49ac-9e9d-64be3d30878a.png)
难受

**Reply by xiehuijie** [reply]
来我的免登录不限速的下载站看看：https://r68s.geek213.fun

**Reply by shaufo** [reply]
https://r68s.geek213.fun/Lean%E5%9B%BD%E9%99%85%E7%89%88

**Comment by kiong2016** [comment]
R22.6.15 docker版本固件没有dockerman

**Reply by npcXxX** [reply]
这种可以自己下的没必要提了吧

**Comment by shaufo** [comment]
上个版本固件的docker管理界面就很友好，很容易创建。615固件的docke CE不习惯，应该是我自己小白。。。

**Reply by YuyangHou1230** [reply]
docker CE只能看吧 感觉啥也干不了啊

**Reply by yuyi4511** [reply]
可以自己装一个dockerman的面板
https://github.com/lisaac/luci-app-dockerman

**Comment by Sn0r1axi** [comment]
刚刚测试6.15固件，确已查到编译了kmod-usb-net-rtl8152
![image](https://user-images.githubusercontent.com/26668216/174221194-1f949061-fb99-4aec-a7c5-12e45df654dc.png)

在8156B网卡连接后并未出现在openwrt首页，也没有出现在接口的物理设置中（应出现WAN4），
![image](https://user-images.githubusercontent.com/26668216/174220869-7f4f975a-865b-467a-96c3-97535ad2163c.png)


但lsusb后能够看到网卡正确接入。
![7K9X)VL{H(W9 C3H%UZ129S](https://user-images.githubusercontent.com/26668216/174220617-5ea361dd-adb1-4f67-80a0-2d8aa45a72c2.png)

**Comment by stwen3399** [comment]
R22.6.15更新遇到的问题
1、MentoHUST无luci显示，附日志
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: 欢迎使用MentoHUST	版本: 0.3.1
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: Copyright (C) 2009-2010 HustMoon Studio
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: 人到华中大，有甜亦有辣。明德厚学地，求是创新家。
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: Bug report to http://code.google.com/p/mentohust/issues/list
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]:
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: !! 打开libnotify失败，请检查是否已安装该库文件。
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[1]:	br-lan
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[2]:	pppoe-wan
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[3]:	eth0
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[4]:	eth1
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[5]:	eth2
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[8]:	eth3
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[9]:	ifb0
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[10]:	ifb1
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡[11]:	bond0
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ?? 请选择网卡[1-11]: ** 您选择了第[11]块网卡。
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ?? 请输入用户名: ?? 请输入密码: ?? 请选择组播地址(0标准 1锐捷私有 2赛尔): ?? 请选择DHCP方式(0不使用 1二次认证 2认证后 3认证前): ** 用户名:
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 网卡: 	bond0
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 认证超时:	8秒
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 响应间隔:	30秒
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 失败等待:	15秒
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 允许失败:	0次
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 组播地址:	标准
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** DHCP方式:	不使用
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: !! 在网卡bond0上获取IP失败!
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: !! 在网卡bond0上获取子网掩码失败!
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 本机MAC:	a6:bb:b5:c2:cb:ad
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 使用IP:	0.0.0.0
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: ** 子网掩码:	255.255.255.255
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: !! 打开网卡bond0失败: bond0: That device is not up
Sun Jun 19 18:08:06 2022 daemon.info mentohust[5259]: >> 认证已退出。
Sun Jun 19 18:08:06 2022 cron.info crond[5282]: crond (busybox 1.30.1) started, log level 8
Sun Jun 19 18:08:07 2022 cron.info crond[5416]: crond (busybox 1.30.1) started, log level 8
Sun Jun 19 18:08:07 2022 daemon.notice procd: /etc/rc.d/S99openclash: OpenClash Already Stop!
Sun Jun 19 18:08:07 2022 daemon.info qbittorrent: Instance "main" disabled.
Sun Jun 19 18:08:07 2022 daemon.notice procd: /etc/rc.d/S99qbittorrent: Command failed: Invalid argument
Sun Jun 19 18:08:07 2022 user.notice nlbwmon: Reloading nlbwmon due to ifup of lan (br-lan)
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5230] NOTICE: ttyd 1.5.2-815bb96c (libwebsockets 3.1.0)
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5231] NOTICE: tty configuration:
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5231] NOTICE:   start command: /bin/login
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5231] NOTICE:   close signal: SIGHUP (1)
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5232] NOTICE:   terminal type: xterm-256color
Sun Jun 19 18:08:07 2022 daemon.err ttyd[5773]: [2022/06/19 18:08:07:5233] NOTICE: Creating Vhost 'default' port 7681, 2 protocols, IPv6 off
Sun Jun 19 18:08:07 2022 daemon.info urandom_seed[5792]: Seed saved (/etc/urandom.seed)
Sun Jun 19 18:08:08 2022 daemon.info dnsmasq[5913]: Connected to system UBus
Sun Jun 19 18:10:25 2022 daemon.err ttyd[5773]: [2022/06/19 18:10:25:1559] NOTICE: time discontiguity: at old time 1655633288s, new time 1655633425s: +137s
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]:
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: 欢迎使用MentoHUST	版本: 0.3.1
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: Copyright (C) 2009-2010 HustMoon Studio
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: 人到华中大，有甜亦有辣。明德厚学地，求是创新家。
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: Bug report to http://code.google.com/p/mentohust/issues/list
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]:
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: !! 打开libnotify失败，请检查是否已安装该库文件。
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[1]:	br-lan
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[2]:	pppoe-wan
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[3]:	eth0
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[4]:	eth1
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[5]:	eth2
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[8]:	eth3
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[9]:	ifb0
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[10]:	ifb1
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡[11]:	bond0
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ?? 请选择网卡[1-11]: ** 您选择了第[11]块网卡。
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ?? 请输入用户名: ?? 请输入密码: ?? 请选择组播地址(0标准 1锐捷私有 2赛尔): ?? 请选择DHCP方式(0不使用 1二次认证 2认证后 3认证前): ** 用户名:
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 网卡: 	bond0
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 认证超时:	8秒
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 响应间隔:	30秒
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 失败等待:	15秒
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 允许失败:	0次
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 组播地址:	标准
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** DHCP方式:	不使用
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: !! 在网卡bond0上获取IP失败!
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: !! 在网卡bond0上获取子网掩码失败!
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 本机MAC:	a6:bb:b5:c2:cb:ad
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 使用IP:	0.0.0.0
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: ** 子网掩码:	255.255.255.255
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: !! 打开网卡bond0失败: bond0: That device is not up
Sun Jun 19 18:10:28 2022 daemon.info mentohust[5951]: >> 认证已退出。
Sun Jun 19 18:10:29 2022 daemon.notice procd: /etc/rc.d/S99zerotier: disabled in config
Sun Jun 19 18:10:29 2022 daemon.info procd: - init complete -
Sun Jun 19 18:10:33 2022 daemon.info mentohust[6267]:
2、路由器开启（AP）模式，有的品牌路由器会抢DHCP，导致LAN路由器正常，LAN口无DHCP服务器，开启如图，没复现过该问题
![image](https://user-images.githubusercontent.com/107385695/174518580-c783aea8-63b1-46e9-9ed4-aaf84079a622.png)

**Reply by smomop** [reply]
我也遇到这个问题，而且保存应用按钮无法生效，被迫回到6.11，感觉这版本有点问题

**Reply by ZaberKo** [reply]
不会是TP的路由器吧? TP的AP模式有个“特性”, 它还是开启DHCP的, 只是监测到上游存在其他DHCP server的信息会转发

**Comment by wenlin5945** [comment]
6.26固件的界面还是docke CE吗，还是和6.11一样的docker界面

**Reply by yuyi4511** [reply]
这个版本 没集成docker

**Comment by Sn0r1axi** [comment]
内置samba以及luci-samba版本请求更新到samba4，否则苹果设备无法拷贝文件至samba服务器。报错OOS 100093

查询到原因是samba3不支持这些新特性： vfs objects = catia fruit streams_xatt

**Comment by Suyuan406** [comment]
雕哥是吧docker版本的忘了吗？

**Comment by yuyi4511** [comment]
aliyundrive-fuse 挂载失败
mkdir -p /mnt/aliyundrive /var/run/aliyundrive-fuse
aliyundrive-fuse -r your-refresh-token -w /var/run/aliyundrive-fuse /mnt/aliyundrive

Error: No such file or directory (os error 2)

**Comment by puteulanus** [comment]
![DA36EFC9-D546-4CFB-AA36-3C538967C75F](https://user-images.githubusercontent.com/4849177/177028403-ce6a097c-0a19-4e66-9689-41e62880835d.png)

R22.7.1 ipset 使用默认软件源更新之后无法使用，提示 `ipset_xlate_argv: symbol not found`

**Comment by yangqiceng** [comment]
@coolsnowwolf 现在是只在QQ群里发布不在这儿更新了么
